src/main/java/frc/trigon/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -0,0 +1,20 @@
+package frc.trigon.robot.subsystems.shooter;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+public class ShooterConstants {
+
+    private static final int LEFT_MOTOR_ID = 3;
+    private static final int RIGHT_MOTOR_ID = 4;
```

Comment:
put right motor above left and switch the ids. right is master so should be higher and have lower id

Reply:
left before right, don't even start

Reply:
Right before left. i will start